### PR TITLE
Filter name for cutflow approach and environment variables

### PR DESCRIPTION
**Pull request by elenavernazza**

### CODE REVIEW

File: python/HHDNN.py [hunk 1]
```diff
@@ -159,29 +159,29 @@ def __init__(self, isZZAnalysis, *args, **kwargs):
             ROOT.gROOT.ProcessLine(".L {}/interface/HHDNNinterface.h".format(base))
             ROOT.gROOT.ProcessLine(".L {}/interface/lester_mt2_bisect.h".format(base))
 
-        if not os.getenv("_HHbbttDNNInput"):
-            os.environ["_HHbbttDNNInput"] = "HHbbttDNNInput"
+        feat_file = "{}/{}/src/cms_runII_dnn_models/models/arc_checks/zz_bbtt/2021-11-22-0/" \
```
Comment (jaimeleonh) [inline python/HHDNN.py]: Hi Elena, this file seems to be very ZZ->bbtt oriented, is it also used for HH->bbtt?

File: python/HHJets.py [hunk 2]
```diff
@@ -248,34 +248,33 @@ def __init__(self, df_filter, *args, **kwargs):
...
         self.df_filter = df_filter
 
+        if "/libToolsTools.so" not in ROOT.gSystem.GetLibraries():
+            ROOT.gSystem.Load("libToolsTools.so")
+        if os.path.expandvars("$CMT_SCRAM_ARCH") == "slc7_amd64_gcc10":
+            ROOT.gROOT.ProcessLine(".include /cvmfs/cms.cern.ch/slc7_amd64_gcc10/"
+                "external/eigen/d812f411c3f9-cms/include/")
+            ROOT.gROOT.ProcessLine(".include /cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/"
+                "tensorflow/2.5.0/include/")
+        elif s.path.expandvars("$CMT_SCRAM_ARCH") == "slc7_amd64_gcc820":
```
Comment (jaimeleonh) [inline python/HHJets.py]: `os.path.expandvars("$CMT_SCRAM_ARCH") == "slc7_amd64_gcc820"`

File: python/HHJets.py [hunk 3]
```diff
@@ -248,34 +248,33 @@ def __init__(self, df_filter, *args, **kwargs):
 
         self.df_filter = df_filter
 
+        if "/libToolsTools.so" not in ROOT.gSystem.GetLibraries():
```
Comment (jaimeleonh) [inline python/HHJets.py]: Why this piece of code needs to be outside of the if block starting in `if not os.getenv("_HHJets"):` as it was before?

Comment (elenavernazza) [reply python/HHJets.py]: Correction in [e68bb34](https://github.com/jaimeleonh/InferenceTools/pull/4/commits/e68bb3433f748ba48fd6ed12a86bc6258f44c5b9)

**Pull request closed without merging**